test/gbdt/test_gbdt.py

```diff
@@ -76,7 +76,19 @@ def test_gbdt_with_save_load(gbdt_cls, stypes, task_type_and_metric):
     loaded_score = loaded_gbdt.compute_metric(dataset.tensor_frame.y, pred)
     dataset.tensor_frame.y = None
     loaded_pred = loaded_gbdt.predict(tf_test=dataset.tensor_frame)
+    # TODO: support more stypes
+    num_features = 0
+    for x in stypes:
+        if x == stype.numerical:
+            num_features += 3 * 1
```

Comment:
I don't quite get the code here.

Reply:
Here I want to get the total number of `FakeDataset` features. 3 means the number and 1 means the dimension.

Reply:
Can you do so by some more generic ways, e.g. getting the values and dimensions from `col_names_dict` or `tensor_frame`, rather than using magic numbers?